styles/components/Internship/CommitteeInternshipInfo.module.scss

```diff
@@ -0,0 +1,63 @@
+@import '../../variables.scss';
+
+$committees: ('ai':$ai-arctic, 'board':$acm-cobalt, 'cyber':$cyber-amber, 'design':$design-orange, 'hack':$hack-purple, 'w':$w-teal, 'icpc':$icpc-tangerine, 'teachla':$teachla-green, 'studio':$studio-raspberry);
+
+.committee-internship-info {
+  background-color: #FFF;
+  border-radius: 1rem;
+  font-size: 1.25em;
+  margin: auto;
+  margin-bottom: 50px;
+  padding: 1.125em 1.5em;
+  position: relative;
+  width: 80vw;
+  
+  .internship-info-data {
+    font-size: 0.8em;
+    line-height: 150%;
+    margin: 0 22px 14px;
```

Comment:
I personally dislike the three field `margin`, since it's confusing. Can we enumerate all four here?